### DISCUSSION TITLE
🚨 📣 We moved to Mattermost 📣 🚨

### DISCUSSION THREAD

**Discussion by MGomezN** [fatiando/community#129] [Announcements]
Hi everyone! As part of Software Underground's move to Mattermost, the #fatiando Slack channel will also move over there. We are happy to see the move of our community to a self-hosted open-source alternative! Huge thanks to everyone who was involved in the migration.
What will happen with this channel?
From now on we'll move our conversations to the Fatiando channel in Swung's Mattermost instance: https://mattermost.softwareunderground.org/

**I'm using the Matrix/Element channel, what's happening with that one?**
We didn't have the time to explore a bridge between the new Mattermost instance and Matrix, so we invite anyone on Matrix to join the Mattermost instance. The conversations will be kept in Mattermost for now.

**Are there any other ways to keep in touch with the community?**
Sure! If you have questions you can open a conversation in this GitHub forum
If you want to report a bug, request a new feature, or contribute with new code, feel free to open an Issue or a Pull Request in any of our GitHub repos: https://github.com/fatiando

Hope to see you all in Mattermost! :wave:

**Comment by santisoler** [comment]
Thanks @MGomezN for the announcement! I pinned it and changed the title. Feel free to change it!
